care/emr/migrations/0056_monetary_components_float_to_decimal.py

```
@@ -0,0 +1,264 @@
+                continue
+            components = []
+            for component in ci.unit_price_components:
+                components.append(forward_monetary_component(component))
+            ci.unit_price_components = components
+            sync_charge_item_costs(ci)
+            bulk_update_list.append(ci)
+        ChargeItem.objects.bulk_update(bulk_update_list, ["unit_price_components"])
+    print("Done: Update ChargeItem unit_price_components to decimal")
+
+
+def reverse_update_charge_item_price_components(apps, schema_editor):
```

Ruff (0.14.11)

Unused function argument: `schema_editor`

(ARG001)